README.yaml

```
@@ -121,18 +121,28 @@ introduction: |-
 
 # How to use this project
 usage: |-
-  Use the `terraform-root-modules` Docker image as the base image in the application `Dockerfile`, and copy the modules from `/aws` folder into `/conf` folder.
-
-  ```dockerfile
-  FROM cloudposse/terraform-root-modules:0.3.2 as terraform-root-modules
-
-  FROM cloudposse/geodesic:0.9.18
-
-  # Copy root modules into /conf folder
-  COPY --from=terraform-root-modules /aws/ /conf/
-
-  WORKDIR /conf/
-  ```
+  With the CloudPosse [reference architecture](https://github.com/cloudposse/reference-architectures), each environment (e.g. staging and prod) has its own, customized
+  version of [Geodesic](https://github.com/cloudposse/geodesic). The process for using one of these Terraform root modules
+  is one of further customizing the environment's Geodesic as follows:
+  1. Create a directory under `/conf` to contain the files related to the root module. Normally the directory has the same name
+  as the root module.
+  1. Copy the `.envrc`, `Makefile.tasks`, and `terraform.envrc` files
+  from [`cloudposse/reference-architectures/templates/conf/kops`](https://github.com/cloudposse/reference-architectures/tree/0.13.0/templates/conf/kops)
```

```suggestion
  from [`cloudposse/reference-architectures/templates/conf/kops`](https://github.com/cloudposse/reference-architectures/tree/master/templates/conf/kops)
```

better to use `master`

---

I thought we did not want to reference master. @osterman ???